v2/robotmk/parse_xml.py

```diff
@@ -0,0 +1,69 @@
+import enum
```

Comment:
Can you briefly explain why we need this? I thought that RF offers parsing functionality for the XML data. Is the functionality offered by RF insufficient for us?

Reply:
:+1: